docs/_sources/routes/direct-storeandnotify.rst.txt

```diff
@@ -0,0 +1,55 @@
+Direct Store and Notify
+***********************
+
+Purpose
+========
+The Linux for Health (LFH) direct:storeAndNotify route allows you to store transaction data as part of the LFH Longitudinal Patient Record (LPR) and notify listeners of the stored data via NATS for downstream integration.
+
+Details
+=======
++-------------------------+---------------------------------------------------------------------+
+| Attribute               | Value                                                               |
++=========================+=====================================================================+
+| Route Name              | direct:storeAndNotify                                               |
++-------------------------+---------------------------------------------------------------------+
+| Supported Operation(s)  | Camel direct invocation                                             |
++-------------------------+---------------------------------------------------------------------+
+
+Configuration
+=============
+The direct:storeAndNotify route expects the following input state in the Camel message Exchange.
+
+Exchange State
+--------------
++-------------------------+--------------------------------------------------------------------+
+| Element                 | Description                                                        |
++=========================+====================================================================+
+| Message Properties      | |msgconfig_def|                                                    |
++-------------------------+---------------+----------------------------------------------------+
+| Message Body            | |msgbody_def|                                                      |
++-------------------------+---------------+----------------------------------------------------+
+
+.. |msgconfig_def| replace:: LFH message properties are stored as Exchange properties.  Before direct:storeAndNotify is called, these Exchange properties must be set via the LinuxForHealthMessage processor. This ensures that the message will be stored in the data store using the LFH standard format.
```

Comment:
The exchange properties are set by the MetadataProcessor. We can change the name of the class if needed for clarity and to align with the docs.

Reply:
I'll update the docs to match the code.  Another good catch.